jf_agent/main.py

```diff
@@ -177,48 +177,46 @@ def load_and_dump_bb(outdir, bb_config, bb_conn, pull_since, pull_until):
     strip_text_content = bb_config.get('strip_text_content', False)
     redact_names_and_urls = bb_config.get('redact_names_and_urls', False)
 
-    write_file(outdir, 'bb_users', get_all_users(bb_conn))
+    users = get_all_users(bb_conn)
+    write_file(outdir, 'bb_users', users)
 
-    project_data = get_all_projects(
-        bb_conn, include_projects, exclude_projects, redact_names_and_urls
+    api_projects, projects = zip(
+        *get_all_projects(bb_conn, include_projects, exclude_projects, redact_names_and_urls)
```

Comment:
Might add a comment as to what this is doing, since the `zip` + spread operator isn't syntax everyone who might touch this code knows